python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -248,6 +251,7 @@ def conv2d(attrs, args):
     return True
 
 
+# conv2d does not need padding check in 20.05
```

Comment:
I cannot link this comment with the code.

Reply:
removed